contrib/mesos/pkg/executor/executor_test.go

```diff
@@ -413,16 +413,34 @@ func TestExecutorLaunchAndKillTask(t *testing.T) {
 // its state.  When a Kamikaze message is received, the executor should
 // attempt suicide.
 func TestExecutorFrameworkMessage(t *testing.T) {
-	t.SkipNow() // TODO(jdef) see comment below re: TASK_FAILED
+	// create fake apiserver
+	podListWatch := NewMockPodsListWatch(api.PodList{})
+	testApiServer := NewTestServer(t, api.NamespaceDefault, &podListWatch.list)
+	defer testApiServer.server.Close()
+
+	// create and start executor
 	mockDriver := &MockExecutorDriver{}
 	kubeletFinished := make(chan struct{})
 	config := Config{
 		Docker:  dockertools.ConnectToDockerOrDie("fake://"),
 		Updates: make(chan interface{}, 1024),
 		APIClient: client.NewOrDie(&client.Config{
-			Host:    "fakehost",
+			Host:    testApiServer.server.URL,
 			Version: testapi.Version(),
 		}),
+		PodStatusFunc: func(kl *kubelet.Kubelet, pod *api.Pod) (*api.PodStatus, error) {
+			return &api.PodStatus{
+				ContainerStatuses: []api.ContainerStatus{
+					{
+						Name: "foo",
```

Comment:
Other executor tests would fail because the status here is for container "foo" but NewTestPod didn't actually name the container anything, so once the tests ran against the latest upstream master they would fail because of the new PodStatusFunc implementation.

I just rebased mesosphere/kubernetes@master to the latest upstream. Could you re-run this test and make sure it doesn't fail similarly?